os/drivers/audio/alc5658.c

```diff
@@ -614,21 +616,25 @@ static int alc5658_getcaps(FAR struct audio_lowerhalf_s *dev, int type, FAR stru
 		}
 
 		switch (caps->ac_format.hw) {
+#ifndef CONFIG_AUDIO_EXCLUDE_SPEAKER_VOLUME
```

Comment:
If both of these enabled, then it will come to default.
so, below is correct, I think
```
switch (caps->ac_format.hw) {
case AUDIO_FU_VOLUME :
#ifndef CONFIG_AUDIO_EXCLUDE_SPEAKER_VOLUME
       caps->....
       caps->....
       break;
#endif
        return -ENOSYS;
```

Reply:
Modified.